src/components/Header/Index.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import * as S from '@/components/Header/Header.styled';
+import logo from '@/assets/images/sumtimeRowLogo.png';
+import { useSession, signOut } from 'next-auth/react';
+import signOutIcon from '@/assets/icons/signOut.svg';
+import { useRouter } from 'next/navigation';
+
+function Header() {
+  const { data: session } = useSession();
+  const router = useRouter();
+
+  return (
+    <S.Container>
+      <S.Logo src={logo.src} alt="logo" onClick={() => router.push('/')} />
+
+      <S.ProfileDiv $align="center">
+        <S.UserName>{session?.user.name}</S.UserName>
+        <S.UserP>님의 하루를 더해보세요!</S.UserP>
+        <S.SignOutBtn onClick={() => signOut({ callbackUrl: '/' })}>
```

Comment:
```suggestion
        <S.SignOutButton onClick={() => signOut({ callbackUrl: '/' })}>
```
P4: 사실 약어를 쓰는 것을 그렇게 선호하지는 하지는 않습니다..! 
개인적인 의견이라 조금 조심스럽긴 합니다... 
누군가는 풀 네임으로 쓰고 누군가는 줄이고 하는 일이 발생할 수도 있을 것이란 생각이 드네요.. 따라서 그냥 다 풀네임으로 써주는 것은 어떤가?! 라는 의견입니다!! 가볍게 읽어주시면 감사하겠습니다~ 😄

Reply:
사실 저도 약어 사용을 고민 했는데, Header에서만 쓰이는 styled component라 약어사용은 괜찮지 않나? 라는 생각으로 사용했던 것 같습니다. 해진님의 말씀처럼 네이밍 관련해서 여러가지 방향이 생겨날 수도 있으니, 약어 관련 내용은 컨벤션으로 논의해보아도 되는 내용인 것 같습니다! 좋은 의견 감사합니다~!